exercises/concept.wip/annalyns-infiltration/.docs/instructions.md

```diff
@@ -29,37 +19,47 @@ julia> can_do_fast_attack(knight_awake)
 false
 ```
 
-_Note: A trailing semicolon suppresses the output of an expression in the top level of the REPL or other interactive contexts (such as Jupyter notebook cells). The output of an expression is only printed by default in the top level of the REPL, not within functions, scripts or other files (unlike what you may expect if you've used Matlab in the past)._ <!-- TODO: Move this to the exercise where the semicolon is first used in examples. Somehow show the Matlab hint only to students who have used Matlab before, when that'll be possible.-->
+<!-- _Note: A trailing semicolon suppresses the output of an expression in the top level of the REPL or other interactive contexts (such as Jupyter notebook cells). The output of an expression is only printed by default in the top level of the REPL, not within functions, scripts or other files._ (unlike what you may expect if you've used Matlab in the past)._ TODO: Move this to the exercise where the semicolon is first used in examples. Somehow show the Matlab hint only to students who have used Matlab before, when that'll be possible.-->
```

Comment:
Why is this commented out?

Reply:
Seemed a bit confusing and not needed for this exercise. If I read the docs correctly, `instructions.md` should be as simple as possible and extra detail like this should be moved somewhere else. Do you agree this is especially important for an exercise so early in the concepts tree?

Not sure where to move it to, though!

Reply:
Students new to Julia will probably be in VS Code (or similar) at this point. We don't provide a Jupyter notebook (should we?)

Reply:
And examples may use it, which is arguably the more important reason.

Reply:
I hadn't noticed that this had become commented. I think we should make it visible again.